modular_chomp/code/modules/mob/living/carbon/human/species/station/traits/negative.dm

```diff
@@ -44,7 +44,7 @@
 /datum/trait/negative/nodefib
 	name = "Unreviveable"
 	desc = "For whatever strange genetic reason, defibs cannot restart your heart."
-	cost = -5
+	cost = -2

```

Comment:
I think this is fair. Now people might be thinking, "Okay but they can just resleeve."

Yes, but you can't be brought back in the field. Your ass has to be dragged all the way back to medical if you forgot your scan, or, you at the very least upon being resleeved have to go _all the way back_ to where you left off. It's still punishing to get killed.

Without this trait, if you get killed in the field, you can still get some hasty medical aid and get defibbed without going home.